invenio_communities/views/ui.py

```diff
@@ -103,10 +103,6 @@ def create_ui_blueprint(app):
         view_func=invitations
     )
 
-    blueprint.add_url_rule(
```

Comment:
https://github.com/inveniosoftware/invenio-communities/blob/19e75547eb113631736c93fd4a1b06dabccdf1ad/invenio_communities/views/communities.py#L243 shouldn't this be deleted as well?